src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import Button from './Button';
+
+function ButtonPanel() {
```

Comment:
- [ ] [OPTIONAL] It will be nice if you use the arrow function as you are using ES6 syntax